services/version-api/src/lambdas/getStatus.ts

```diff
@@ -0,0 +1,66 @@
+import * as response from '../libs/response';
+import config from '../libs/config';
+import params from '../libs/params';
+import log from '../libs/logs';
+
+import { getApplicationUpdateStatus } from '../helpers/getApplicationUpdateStatus';
+
+const USER_AGENT = 'User-Agent';
+const SPLIT_SYMBOL = '/';
+const SUPPORTED_OS = ['ios', 'android'];
+
+type DeviceOS = 'android' | 'ios';
+
+export interface DeviceVersions {
+  min: string;
+  max: string;
+  updateUrl: string;
+}
+
+export interface DeviceConfiguration {
+  versions: Record<DeviceOS, DeviceVersions>;
+}
+
+export interface Dependencies {
+  getVersionConfigurations: (environment: string) => Promise<DeviceConfiguration>;
+}
+
+export interface Headers {
+  'User-Agent': string;
+}
+
+export interface LambdaRequest {
+  headers: Headers;
+}
+
+function isSupportedOs(deviceOS: string): deviceOS is DeviceOS {
+  return SUPPORTED_OS.includes(deviceOS);
+}
+
+export async function getStatus(input: LambdaRequest, dependencies: Dependencies) {
+  const userAgent = input.headers[USER_AGENT];
+  const [, deviceApplicationVersion, deviceOS] = userAgent.split(SPLIT_SYMBOL);
+
+  const isMalformedUserAgent = !isSupportedOs(deviceOS) || !deviceApplicationVersion;
+  if (isMalformedUserAgent) {
+    log.error(
```

Comment:
Change to the updated log.writeError(...)

Reply:
fixed